terraform/applications/moderator/email.tf

```diff
@@ -0,0 +1,115 @@
+#
+# SES Domain & Verification Setup within Moderator Environment AWS Zone
+#
+resource "aws_ses_domain_identity" "main" {
+  domain = var.moderator_mozilla
+}
+
+resource "aws_ses_domain_identity_verification" "main" {
+  domain     = aws_ses_domain_identity.main.id
+  depends_on = [aws_route53_record.ses_verification]
+}
+
+resource "aws_route53_record" "ses_verification" {
+  zone_id = module.moderator_mozilla.zone_id
+  name    = "_amazonses.${aws_ses_domain_identity.main.id}"
+  type    = "TXT"
+  ttl     = "600"
+  records = [aws_ses_domain_identity.main.verification_token]
+}
+
+resource "aws_ses_domain_dkim" "main" {
+  domain = aws_ses_domain_identity.main.domain
+}
+
+resource "aws_route53_record" "dkim" {
+  count   = 3
```

Comment:
Why `3`, is that our zone count? Can we use a terraform output here to get the zone length instead of hardcoding?